### PR TITLE
fix: #425 Price range filter

### PR DESCRIPTION
**Pull request by strrife**
### Please check if the PR fulfills these requirements ###

- [+] The commit message follows our guidelines

**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Fix for https://github.com/deity-io/falcon/issues/425

**What is the current behavior? (You can also link to an open issue here)**

Now the price filter does not work at all

**What is the new behavior (if this is a feature change)?**

Price filter works, except for cases described in the issue I am creating right now

**Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)**

Basically the problem is with the app itself rather than packages, so to fix this in an already existing store one would have to fix the filter logic.

### CODE REVIEW

**Comment by tutok** [comment]
@strrife thanks for your contribution!
